src/main/java/frc/robot/subsystems/arm/C2025ArmIO.java

```diff
@@ -0,0 +1,120 @@
+package frc.robot.subsystems.arm;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.controls.NeutralOut;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.system.plant.LinearSystemId;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import frc.robot.Robot;
+
+public class C2025ArmIO implements ArmIO {
+    private final TalonFX shoulderMotor;
+    private final TalonFX wristMotor;
+
+    private final DCMotorSim shoulderSim = new DCMotorSim(
+            LinearSystemId.createDCMotorSystem(DCMotor.getKrakenX60(1),
+                    0.001, 1.0),
+            DCMotor.getKrakenX60(1).withReduction(1.0));
+
+    private final DCMotorSim wristSim = new DCMotorSim(
+            LinearSystemId.createDCMotorSystem(DCMotor.getKrakenX60(1),
+                    0.001, 1.0),
+            DCMotor.getKrakenX60(1).withReduction(1.0));
+
+    private final MotionMagicVoltage positionRequest = new MotionMagicVoltage(0.0);
+    private final NeutralOut stopRequest = new NeutralOut();
+
+    public C2025ArmIO(TalonFX shoulderMotor, TalonFX wristMotor) {
+        this.shoulderMotor = shoulderMotor;
+        this.wristMotor = wristMotor;
+
+        TalonFXConfiguration config = new TalonFXConfiguration();
+        config.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+
+        config.Slot0.kV = 0.115;
+        config.Slot0.kA = 0.01;
+        config.Slot0.kP = 0.5;
+        config.Slot0.kI = 0.0;
+        config.Slot0.kD = 0.0;
+
+        config.MotionMagic.MotionMagicAcceleration = 2500.0 / 60.0;
+        config.MotionMagic.MotionMagicCruiseVelocity = 5500.0 / 60.0;
+
+        shoulderMotor.getConfigurator().apply(config);
+        wristMotor.getConfigurator().apply(config);
```

Comment:
Once we get information about the gear reductions from CAD we'll be splitting the config into two separate configurations. This is fine for now.